src/validation.cpp

```diff
@@ -2023,15 +2025,23 @@ bool CChainState::FlushStateToDisk(
     static int64_t nLastFlush = 0;
     std::set<int> setFilesToPrune;
     bool full_flush_completed = false;
+
+    const size_t coins_count = CoinsTip().GetCacheSize();
+    const size_t coins_mem_usage = CoinsTip().DynamicMemoryUsage();
```

Comment:
How is this different from the existing `cacheSize`?

Reply:
Ah yeah, guess it's not. Oops.